trl/trainer/ppo_trainer.py

```diff
@@ -875,6 +879,17 @@ def create_model_card(self, path: str, model_name: Optional[str] = "TRL Model")
         with open(os.path.join(path, "README.md"), "w", encoding="utf-8") as f:
             f.write(model_card_content)
 
+    def set_seed(seed: int):
```

Comment:
```suggestion
    def set_seed(self, seed: int):
```
This should fix the failing CI tests